### ISSUE TITLE
Lets talk about the Gutenberg editor

### ISSUE THREAD

**Issue by danieltj27** [danieltj27/Dark-Mode#43]
As you may or may not know - [Gutenberg](https://wordpress.org/plugins/gutenberg/) will one day be introduced into Core, replacing not only the TinyMCE editor, but the whole edit post/pages page itself which is quite a huge change. There will be quite a large task ahead of making sure the Gutenberg editor (well, technically it's an entire page overhaul) is redesigned to fit into the Dark Mode styling.

I'm not suggesting that it be incorporated into Dark Mode's standard stylesheet just yet as the plugin is still in beta - but I think it'd be worth making a start so it's a much easier transition. I'm going to make a branch where we can develop the Gutenberg UI separately from the main DM project making things a bit easier. I personally don't feel like there's any rush just yet, but I'd ideally like to think we can have it ready before [5.0](https://make.wordpress.org/core/5.0/) drops (which is when Gutenberg is planned to be merged in). 5.0 hasn't had any dates set just yet (as of the time I write this), but it should in theory have a few months yet.

Please make sure that this issue is referenced in any future PRs too so they're documented.

**Comment by danieltj27** [comment]
Gutenberg branch now [lives here](https://github.com/danieltj27/Dark-Mode/tree/gutenberg). The base commit for this branch is **1.4** at commit 0afc3b9.

**Comment by danieltj27** [comment]
Now we have v1.6 released, next port of call is to bump the Gutenberg branch up to 1.6 and then start working on this a little bit to get things moving.

**Comment by danieltj27** [comment]
See issue #77.

**Comment by surferking** [comment]
Time to re-open this ready for the imminent timeline for this becoming a core feature?

**Comment by danieltj27** [comment]
Not until it's merged into Core. Whilst it's still a plugin lots of things can change.
